RawSpeed/DngDecoderSlices.cpp

```diff
@@ -143,6 +143,159 @@ my_error_throw (j_common_ptr cinfo)
   ThrowRDE("JPEG decoder error!");
 } 
 
+static void decodeFPDeltaRow(unsigned char * src, unsigned char * dst, size_t tileWidth, size_t realTileWidth, unsigned int bytesps, int factor) {
```

Comment:
https://github.com/jcelaya/hdrmerge/issues/103#issuecomment-270953456
So `MIT License, copyright 2014 Javier Celaya <jcelaya@gmail.com>`
Now, can it be used in with LGPL?